apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -99,7 +99,7 @@ type VSHNKeycloakServiceSpec struct {
 	// +kubebuilder:default="/"
 	RelativePath string `json:"relativePath,omitempty"`
 
-	// +kubebuilder:validation:Enum="23";"24"
+	// +kubebuilder:validation:Enum="23";"24";"25"
```

Comment:
I'm not sure if we should even do such a validation here, as we would have to adjust this field on every new release.
Maybe it would make sense to just specify a Minimum with `+kubebuilder:validation:Minimum`?

Reply:
but then we would have to update that minimum with almost every release, time saving is minimal here. Either we disable that or keep like it is imho

Reply:
The best approach would be to do the validation into a webhook. In the webhook we could check direclty what versions are available. In that case no need to track this and update each time there is a new version

Reply:
As we see with this version, there can be potentially breaking changes that make never version non-working at all (eg. changed values in the helm chart).
Therefore we actually should keep this list hardcoded with all versions tested and verified by us (and Inventage)

Reply:
I agree, that was my concern with this from the beginning.